src/components/reviewList.jsx

```diff
@@ -0,0 +1,37 @@
+
+import ReviewCard from "./ReviewCard"
+const ReviewList = (props) => {
+ const {reviewData, setReviewData} = props
+
+let count = 0
```

Comment:
I would rethink this as this could lead to potential bugs, this review has an unique id so you could use that